run.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/sh
+
+# Parse arg if any
+ARGS1=${1:-"./config/config.json"}
+echo $ARGS1
+
+# Main .json configuration file
+config_file=$ARGS1
+
+# Needs JQ to parse the main .json config file: 
+# sudo apt-get install jq
+# brew install jq
+
+enclave_name=$(cat $config_file | jq -r ".enclave_name")
```

Comment:
Regarding passing a single parameter instead multiple command line args I sort of agree it would be more consistent but I did not want to change how the topology script works in this PR. I m going to create an issue to fix that if @0xFugue  agrees

Reply:
Regarding the topology as a service within the enclave, goin to start working on it. Fingers crossed store_service_files works as expected.